src/components/MAPI/apps/ClusterDetailApps.tsx

```diff
@@ -86,15 +87,16 @@ const PreinstalledApps = styled.div`
   }
 `;
 
+// FIXME(axbarsan): Fetch cluster by ID in URL params once AWS has namespaced clusters.
 interface IClusterDetailApps {
-  clusterId: string;
   releaseVersion: string;
 }
 
 const ClusterDetailApps: React.FC<IClusterDetailApps> = ({
-  clusterId,
   releaseVersion,
 }) => {
+  const { clusterId } = useParams<{ clusterId: string }>();
```

Comment:
> // FIXME(axbarsan): Fetch cluster by ID in URL params once AWS has namespaced clusters.

Isn't that happening here?

Reply:
Here we only get the cluster ID from the URL, but we get the cluster's release version as a prop (to maintain compatibility with the GS API world.

What I want to do later on is to fetch the cluster from MAPI here, and get the release version from the cluster itself, and not as a prop.